contracts/payloads/IGP43/PayloadIGP43.sol

```diff
@@ -0,0 +1,643 @@
+    ///                                 This function can only be called by the owner.
+    /// @param deploymentLogic_         The address of the vault deployment logic contract to be set.
+    /// @param allowed_                 A boolean indicating whether the specified address is allowed to deploy new type of vault.
+    function setVaultDeploymentLogic(address deploymentLogic_, bool allowed_) external;
+
+    /// @notice                         Sets an address (`vaultAuth_`) as allowed vault authorization or not for a specific vault (`vault_`).
+    ///                                 This function can only be called by the owner.
+    /// @param vault_                   The address of the vault for which the authorization is being set.
+    /// @param vaultAuth_               The address to be set as vault authorization.
+    /// @param allowed_                 A boolean indicating whether the specified address is allowed to update the specific vault config.
+    function setVaultAuth(address vault_, address vaultAuth_, bool allowed_) external;
+
+    /// @notice                         Computes the address of a vault based on its given ID (`vaultId_`).
+    /// @param vaultId_                 The ID of the vault.
+    /// @return vault_                  Returns the computed address of the vault.
+    function getVaultAddress(
+        uint256 vaultId_
+    ) external view returns (address vault_);
+}
+
+interface FluidDexFactory {
+    /// @notice                         Computes the address of a dex based on its given ID (`dexId_`).
+    /// @param dexId_                   The ID of the dex.
+    /// @return dex_                    Returns the computed address of the dex.
+    function getDexAddress(uint256 dexId_) external view returns (address dex_);
+}
+
+contract PayloadIGP43 {
+
+    uint256 public constant PROPOSAL_ID = 43;
+
+    address public constant PROPOSER =
+        0xA45f7bD6A5Ff45D31aaCE6bCD3d426D9328cea01;
+    address public constant PROPOSER_AVO_MULTISIG =
+        0x059a94a72451c0ae1Cc1cE4bf0Db52421Bbe8210;
+    address public constant PROPOSER_AVO_MULTISIG_2 =
+        0x9efdE135CA4832AbF0408c44c6f5f370eB0f35e8;
+    address public constant PROPOSER_AVO_MULTISIG_3 =
+        0x5C43AAC965ff230AC1cF63e924D0153291D78BaD;
+
+    IGovernorBravo public constant GOVERNOR = IGovernorBravo(0x0204Cd037B2ec03605CFdFe482D8e257C765fA1B);
+    ITimelock public constant TIMELOCK = ITimelock(0x2386DC45AdDed673317eF068992F19421B481F4c);
+
+    address public constant TEAM_MULTISIG = 0x4F6F977aCDD1177DCD81aB83074855EcB9C2D49e;
+
+    address public immutable ADDRESS_THIS;
+
+    IFluidLiquidityAdmin public constant LIQUIDITY = IFluidLiquidityAdmin(0x52Aa899454998Be5b000Ad077a46Bbe360F4e497);
+
+    FluidVaultFactory public constant VAULT_FACTORY = FluidVaultFactory(0x324c5Dc1fC42c7a4D43d92df1eBA58a54d13Bf2d);
+    FluidDexFactory public constant DEX_FACTORY = FluidDexFactory(0x93DD426446B5370F094a1e31f19991AAA6Ac0bE0);
```

Comment:
sorry commented the old factory. this is the new one: 0xF9b539Cd37Fc81bBEA1F078240d16b988BBae073